<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatrolDetectTaskEditorClass</class>
 <widget class="ITaskEditor" name="PatrolDetectTaskEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PatrolDetectTaskEditor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>游曳探测任务</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>246</width>
           <height>743</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>一般属性</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tb_patroDetectlId">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>74</number>
              </attribute>
              <attribute name="horizontalHeaderHighlightSections">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>新建列</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>新建列</string>
               </property>
              </column>
              <item row="0" column="0">
               <property name="text">
                <string>ID</string>
               </property>
              </item>
              <item row="0" column="1">
               <property name="text">
                <string/>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QCheckBox" name="checkBox_2">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>结束条件</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tb_patroDetectEnd">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>122</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>12</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderMinimumSectionSize">
               <number>25</number>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <row>
               <property name="text">
                <string/>
               </property>
              </row>
              <row>
               <property name="text">
                <string/>
               </property>
              </row>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string/>
               </property>
              </row>
              <column>
               <property name="text">
                <string/>
               </property>
              </column>
              <column>
               <property name="text">
                <string/>
               </property>
              </column>
              <item row="0" column="0">
               <property name="text">
                <string>预设结束</string>
               </property>
              </item>
              <item row="0" column="1">
               <property name="text">
                <string/>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </item>
              <item row="1" column="0">
               <property name="text">
                <string>结束条件</string>
               </property>
              </item>
              <item row="2" column="0">
               <property name="text">
                <string>时长</string>
               </property>
              </item>
              <item row="3" column="0">
               <property name="text">
                <string>时刻</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QCheckBox" name="checkBox_3">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>位置</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tb_patroDetectPosition">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>122</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>150</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>74</number>
              </attribute>
              <attribute name="horizontalHeaderHighlightSections">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>新建列</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>新建列</string>
               </property>
              </column>
              <item row="0" column="0">
               <property name="text">
                <string>Z向</string>
               </property>
              </item>
              <item row="1" column="0">
               <property name="text">
                <string>Z向单位</string>
               </property>
              </item>
              <item row="2" column="0">
               <property name="text">
                <string>速度</string>
               </property>
              </item>
              <item row="3" column="0">
               <property name="text">
                <string>速度单位</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QCheckBox" name="checkBox_4">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>任务属性</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tb_patroDetectTask">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>122</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>150</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>74</number>
              </attribute>
              <attribute name="horizontalHeaderHighlightSections">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>新建列</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>新建列</string>
               </property>
              </column>
              <item row="0" column="0">
               <property name="text">
                <string>区域坐标</string>
               </property>
              </item>
              <item row="1" column="0">
               <property name="text">
                <string>游曳类型</string>
               </property>
              </item>
              <item row="2" column="0">
               <property name="text">
                <string>覆盖间隔</string>
               </property>
              </item>
              <item row="3" column="0">
               <property name="text">
                <string>目标处理</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="checkBox_5">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>回传属性</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tb_patroDetectPassBack">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>150</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>74</number>
              </attribute>
              <attribute name="horizontalHeaderHighlightSections">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>新建列</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>新建列</string>
               </property>
              </column>
              <item row="0" column="0">
               <property name="text">
                <string>回传天数</string>
               </property>
              </item>
              <item row="1" column="0">
               <property name="text">
                <string>回传时刻</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QCheckBox" name="checkBox_6">
              <property name="font">
               <font>
                <family>宋体</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>探测手段</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tb_patroDetectWay">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>150</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>宋体</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>74</number>
              </attribute>
              <attribute name="horizontalHeaderHighlightSections">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>新建列</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>新建列</string>
               </property>
              </column>
              <item row="0" column="0">
               <property name="text">
                <string>光电探测</string>
               </property>
              </item>
              <item row="1" column="0">
               <property name="text">
                <string>电磁探测</string>
               </property>
              </item>
              <item row="2" column="0">
               <property name="text">
                <string>舷侧阵启动</string>
               </property>
              </item>
              <item row="3" column="0">
               <property name="text">
                <string>拖曳阵启动</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ITaskEditor</class>
   <extends>QWidget</extends>
   <header>itaskeditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
